### PR TITLE
Adds ability to reload messages and message data after saving broadcast emails.

### PR DESCRIPTION
**Pull request by randy-girard**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # TeamSnap JavaScript SDK CHANGELOG
 
 ### June 1, 2016 // Version 1.13.4
+- Reloads `messages` and `messageData` in `saveBroadcastEmail` method in persistence wrapper.
```
Comment (amberwebb) [inline CHANGELOG.md]: Since this didn't get merged yesterday, should you break this out to be `June 2, 2016 // Version 1.13.5` ?

Comment (randy-girard) [inline CHANGELOG.md]: Yea I will update it when i get rdy to merge

**Comment by amberwebb** [comment]
Once versioning is changed, 👍

**Comment by randy-girard** [comment]
@amberwebb Updated the version. Quick relook at this if you can?

@matthopson Would you be able to release this? or can anyone do it?

**Comment by amberwebb** [comment]
Looks good Randy.

**Comment by matthopson** [comment]
@randy-girard I can release it.. I've got an SDK PR up as well... will merge after you and go from there.

**Comment by randy-girard** [comment]
@matthopson Merged!

**Pull request merged**